library/core/class.cache.php

```diff
@@ -564,9 +570,14 @@ protected function fallback($key, $options) {
 
     public function getPrefix($forcePrefix = null, $withRevision = true) {
         static $configPrefix = false;
+        $cKey = ($forcePrefix ?? 'null:').($withRevision ? 'true' : 'false');
+        if (key_exists($cKey, self::$cConfig)) {
```

Comment:
Shouldn't this be `array_key_exists()`? Looks like this an alias from PHP <5 that's still around.

Reply:
Done.